beacon-chain/blockchain/stategenerator/state_generator_test.go

```diff
@@ -58,8 +57,9 @@ func TestGenerateState_OK(t *testing.T) {
 		t.Fatalf("Unable to save state in chain %v", err)
 	}
 
-	slotToGenerate := genesisSlot + 2*(slotLimit)
-	newState, err := GenerateStateFromSlot(context.Background(), beaconDb, slotToGenerate)
+	inMemBlocks := bd.InMemoryBlocks()
+	blockToGenerateTill := inMemBlocks[len(inMemBlocks)-1]
```

Comment:
This is the last block, is that correct?

Reply:
Yeap